subsys/bootloader/cmake/sign.cmake

```diff
@@ -4,16 +4,14 @@
 # SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
 #
 
-
-#TODO: Detect which elf to depend on and resolve which
-#      image should be hashed and signed by b0.
-
-set(KERNEL_ELF_TO_BE_SIGNED zephyr_final)
+set(signed_hex ${PROJECT_BINARY_DIR}/signed_by_b0_${KERNEL_HEX_NAME})
```

Comment:
Do we want this to be done for s1 as well?

Reply:
Yes, that will come at a later time.